src/main/java/com/bitso/websockets/BitsoWebSocket.java

```diff
@@ -154,7 +154,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg)
 
             if (msg instanceof FullHttpResponse) {
                 FullHttpResponse response = (FullHttpResponse) msg;
-                throw new Exception("Unexpected FullHttpResponse (getStatus=" + response.getStatus() + ", content="
+                throw new Exception("Unexpected FullHttpResponse (getStatus=" + response.status() + ", content="
```

Comment:
Nice catch, please update what I've asked in comments to proceed with your PR merge.